src/Diagram.js

```diff
@@ -247,19 +248,24 @@ class Diagram extends React.PureComponent {
     this.updateScroll(e.currentTarget);
   };
 
+  changeScrollWithMinimap = pos => {
```

Comment:
`scrollToPosition` <> `changeScrollWithMinimap`

Reply:
will do this in the next commit

Reply:
done